pkg/operator/ceph/cluster/osd/topology/topology_test.go

```diff
@@ -154,3 +155,153 @@ func TestGetDefaultTopologyLabels(t *testing.T) {
 		"topology.rook.io/datacenter"
 	assert.Equal(t, expectedLabels, GetDefaultTopologyLabels())
 }
+
+func TestCheckTopologyConflicts(t *testing.T) {
```

Comment:
An interesting test case:

```
region          region1
                   |
zone             r1-z1
               /        \
rack    r1-z1-r1        <empty>
          |                |
node    r1-z1-r1-n1    r1-z1-r2-n2
```

What do we want to happen if a label is missing from one part of the map?